libs/utils/src/sentry_init.rs

```diff
@@ -0,0 +1,17 @@
+pub fn init_sentry(maybe_sentry_url: Option<&String>, process_name: &str) {
+    match maybe_sentry_url {
+        Some(sentry_url) => {
+            let _guard = sentry::init((
```

Comment:
the lifetime of this `_guard` is not too long to catch all everything. Should it be used in `main`?

Reply:
I'll also recommend to mark this method with `#[must_use]` attribute to generate warning for everyone who will forget to use the return value

Reply:
Nit: if it's used, it should be `guard`, not `_guard`.

Reply:
Also, maybe add a comment to describe what to do with the returned value: keep it alive while something-something-something.

Reply:
So it seems to me that _guard doesn't need to be used. I'm keeping it more as a stylistic copy of the sentry tutorial. It also seems to me that it doesn't need to be kept alive either (but I'm less sure about this).

Reply:
https://docs.rs/sentry/latest/sentry/fn.init.html

> This returns a client init guard that must be kept in scope and that will help the client send events before the application closes. When the guard is dropped, then the transport that was initialized shuts down and no further events can be sent on it.

Seems like we _have to_ keep it alive, otherwise Sentry reporting won't work.

Reply:
Ah, by "used" I meant "returned from the function". We use it inside the function by returning it.

Reply:
thanks! Since my editor likes _guard more (since it's not explicitly used) I'll keep it as such.